website/docs/wallet/exiting-a-validator.md

```diff
@@ -0,0 +1,39 @@
+---
+id: exiting-a-validator
+title: Exiting a validator
+sidebar_label: Exiting a validator
+---
+
+## Performing a voluntary exit
+
+You can exit one or more validator accounts in your wallet (HD, non-HD or remote) using the following command:
```

Comment:
Let's avoid using `you` in documentation, according to @har00ga, as a best practice. Let's use `One can exit`